harper-dictionary-parsing/src/span.rs

```diff
@@ -11,9 +11,7 @@ pub struct Span {
 
```

Comment:
I don't love the idea of putting `Span` in this crate. It feel somewhat arbitrary. Would it be possible to put it back in `harper-core` or perhaps a new crate?

Reply:
Putting it in a new crate is probably best, but we don't want to make too many crates! Maybe `harper-lib`?

Reply:
Did that.